examples/private_set_intersection/main_module/rust/src/lib.rs

```diff
@@ -81,6 +82,7 @@ impl oak::WithInit for Router {
             LogInit {
                 log_sender: Some(log_sender),
             },
+            true,
```

Comment:
This is the only place where the `keep_sender_open` flag I added needs to be `true`, and I am not sure why. 
Changing this to `false` results in an orphaned channel (`handler-in(5)`). 
<img width="778" alt="Screen Shot 2021-02-10 at 18 59 25" src="https://user-images.githubusercontent.com/12942581/107558046-3fa4c800-6bd2-11eb-99d7-46348d01fefe.png">
I don't understand why this happens. I expected the following line to close the read-half of the `handler-in` channel. 
https://github.com/project-oak/oak/blob/4da10691dfe867a022d2d37c7b991d5bdaaf3673/sdk/rust/oak/src/lib.rs#L754